build.py

```diff
@@ -65,13 +65,13 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.12.0':
-      ('21.07',   # triton container
-       '21.07',      # upstream container
-       '1.8.0',      # ORT
-       '2021.2.200', # ORT OpenVINO
-       '2021.2',     # Standalone OpenVINO
-       '2.2.8')      # DCGM version
+    '2.12.0': (
+        '21.07',  # triton container
+        '21.07',  # upstream container
+        '1.8.0',  # ORT
+        '2021.2.200',  # ORT OpenVINO
+        '2021.2',  # Standalone OpenVINO
+        '2.2.3')  # DCGM version
```

Comment:
We can move forward to 2.2.8 for 21.08 so you can close https://github.com/triton-inference-server/server/pull/3143